packages/adblocker-extended-selectors/src/eval.ts

```diff
@@ -363,6 +411,10 @@ export function querySelectorAll(element: Element, selector: AST): Element[] {
   }
 
   if (selector.type === 'pseudo-class') {
+    if (selector.name === 'xpath') {
+      return handleXPathSelector(element, selector.argument);
+    }
```

Comment:
This should be touched via another feature fix. See [`7126a61` (#4974)](https://github.com/ghostery/adblocker/pull/4974/commits/7126a61a54e3435b9f2d9de085bfe7ec6d56ba8a).